modules/tensor_mechanics/test/tests/critical_time_step/crit_time_solid_variable.i

```diff
@@ -0,0 +1,83 @@
+[GlobalParams]
+  displacements = 'disp_x'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 50
+
+  xmin = 0
+  xmax = 5
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+  [../]
+[]
+
+[BCs]
+  [./2_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 1
+    value = 0.0
+  [../]
+[]
+
+[Functions]
+  [./prefac]
+    type = ParsedFunction
+    value = '1+2*x'
+  [../]
+[]
+
+[Materials]
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    poissons_ratio = 0.1
+    youngs_modulus = 1e6
+    elasticity_tensor_prefactor = prefac
+  [../]
+  [./strain]
+    type = ComputeSmallStrain
+  [../]
+  [./stress]
+    type = ComputeLinearElasticStress
+  [../]
+  [./density]
+    type = GenericConstantMaterial
+    prop_names = 'density'
+    prop_values = '8050.0'
+  [../]
+[]
+
+[Executioner]
+  type = Transient
```

Comment:
Shouldn't you be using your CriticalTimeStep postprocessor to determine timestep size? With the way these blocks are setup, isn't it just constant dt?